plugins/modules/cohesity_oracle_job.py

```diff
@@ -527,7 +527,7 @@ def main():
         check__mandatory__params(module)
         body = ProtectionJobRequestBody()
         body.name = module.params.get("name")
-        body.parent_source_id = source_id
+        body.parent_source_id = parent_id
```

Comment:
This will not affect other adapters correct ?

Reply:
This is oracle specific module, this change wont affect other adapters.